Makefile

```diff
@@ -76,7 +76,7 @@ docker:
 	buf format proto -w
 
 #	check for breaking changes against master branch
-	buf breaking proto --against ".git#branch=master,subdir=proto"
+#   buf breaking proto --against ".git#branch=master,subdir=proto"
```

Comment:
can we uncomment this line once the protofiles are generated, or do we have to do a separate PR?